src/Setup.php

```diff
@@ -0,0 +1,218 @@
+<?php
+/**
+ * Initialization of theme.
+ *
+ * @since 1.0.0
+ *
+ * @package Bcgov/Theme/Block
+ */
+
+namespace Bcgov\Theme\Block;
+
+/**
+ * Setup class setups up theme.
+ */
+class Setup {
+
+    /**
+     * Constructor.
+     */
+    public function __construct() {
+        $this->hook();
+    }
+
+    /**
+     * Sets up hooks for theme.
+     *
+     * @return void
+     */
+    private function hook() {
+        $loader = new Loader();
+		$loader->add_action( 'after_setup_theme', $this, 'bcgov_block_theme' );
```

Comment:
```suggestion
		add_action( 'after_setup_theme',[ $this, 'bcgov_block_theme' ]);
```
repeat, instead of using the loader.